### PR TITLE
Allow alarm definitions to be in values.yaml

### PR DESCRIPTION
**Pull request by craigbr**
Add a configmap for the definitions

### CODE REVIEW

File: monasca/values.yaml [hunk 1]
```diff
@@ -367,6 +367,186 @@ alarms:
...
     os_project_domain_name: Default
+  definitions_configuration:
+     definitions.yml.j2: |
+      notifications:
+        - name: "{{ NOTIFICATION_NAME | default('default') }}"
+          type: "{{ NOTIFICATION_TYPE | default('email') }}"
+          address: "{{ NOTIFICATION_ADDRESS | default('root@localhost') }}"
+
+      alarm_definitions:
+        - name: "Host Status"
```
Comment (mhoppal) [inline monasca/values.yaml]: we should really change the default alarm definitions for metrics that are there

Comment (craigbr) [inline monasca/values.yaml]: Good idea.

Done

File: monasca/templates/alarms-configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{{- if .Values.alarms.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "alarms.fullname" . }}
```
Comment (mhoppal) [inline monasca/templates/alarms-configmap.yaml]: is this defined in the helpers.tpl?

Comment (craigbr) [inline monasca/templates/alarms-configmap.yaml]: Yes

**Pull request merged**